scanners/sslyze/parser/parser.test.js

```diff
@@ -396,10 +422,10 @@ test("parses result file for self-signed.badssl.com correctly", async () => {
     reference: null,
   });
 });
-
+/*
```

Comment:
Commented-out test => can this be re-added or deleted?

Reply:
Yes. I just readded it